src/Runner.Worker/ActionRunner.cs

```diff
@@ -165,14 +164,14 @@ public async Task RunAsync()
                 stepHost = containerStepHost;
             }
 
+
             // Setup File Command Manager
             var fileCommandManager = HostContext.CreateService<IFileCommandManager>();
             fileCommandManager.InitializeFiles(ExecutionContext, null);
 
             // Load the inputs.
             ExecutionContext.Debug("Loading inputs");
-            var templateEvaluator = ExecutionContext.ToPipelineTemplateEvaluator();
-            var inputs = templateEvaluator.EvaluateStepInputs(Action.Inputs, ExecutionContext.ExpressionValues, ExecutionContext.ExpressionFunctions);
+            var inputs = EvaluateStepInputs(stepHost);
```

Comment:
I would prefer to wrap this change behind a feature flag so we can easily roll back. Breaking inputs are not fun.

Reply:
[Feature flag example here](https://github.com/actions/runner/blob/909b05eb66a81e278a6f95e80532f972bacaf2b0/src/Runner.Worker/Handlers/NodeScriptActionHandler.cs#L117)